src/components/KSkeleton/KSkeleton.vue

```diff
@@ -1,8 +1,8 @@
 <template>
   <div
     v-if="isVisible"
-    class="d-flex flex-wrap k-skeleton-container"
-    :class="{ 'w-100': type !== 'spinner', 'opacity-0': !isVisible }"
```

Comment:
We don't need opacity css here anymore?

Reply:
We have a `v-if="isVisible"` on that element. So when `isVisible` (which is a prop) is false - the element won't render - which means that class will never apply. Now that I look at it again, I will add a commend to remove `isVisible` prop in the breaking change.